### DISCUSSION TITLE
Cool Ideas and Challenges

### DISCUSSION THREAD

**Discussion by jurjendevries** [bsiever/microbit-pxt-blehid#26] [General]
Let's inspire each other with the Cool Ideas and Challenges you have made with the blehid extension.

**Comment by jurjendevries** [comment]
To start this topic I like to share the DIY MaKey MaKey by a microbit.

With this little piece of hardware you can connect your computer to everything in the world that is conductive. Playing a piano by a banana? Flappy Bird flying by you jumping in a box of water? It's all possible

![PXL_20230227_170618131](https://user-images.githubusercontent.com/355488/221656269-de523562-a6f0-4507-8337-d94ad9f50057.jpg)

Do you want to create one yourself by a microbit? Have a look at my demo and code by https://gitlab.com/jurjendevries/makey-makey-microbit

Thank you @bsiever for your great effort making this extension.

**Reply by bsiever** [reply]
Great example!  Thanks for sharing!

Here are some of the examples I've created:  https://youtu.be/n4J5GN72N_4 .  None are as fun as playing a MaKey game with a bowl of fruit though...

By the way, your writeup mentions micro:bit v1 or v2 --- this extension only works on the v2 for now. 

Thanks again!

**Comment by 4-R-Forum** [comment]
# Glove Flies Earth using [microbit-pxt-blehid](https://github.com/bsiever/microbit-pxt-blehid)

![FlyEarth7-Pyramids-Glove](https://github.com/user-attachments/assets/1aa1dc6b-4119-4a72-969a-dad8f1959b36)
Code and documentation at https://github.com/4-R-Forum/flyearth-07


See Instructable and demo at  https://www.instructables.com/Explore-Earth-Fly-With-a-Turn-of-Your-Hand-and-Dis/

**Reply by bsiever** [reply]
Thanks for sharing!
